doozer/doozerlib/cli/olm_bundle.py

```diff
@@ -231,6 +231,11 @@ def rebase_and_build_olm_bundle(runtime: Runtime, operator_nvrs: Tuple[str, ...]
     """
 
     runtime.initialize(config_only=True)
+    if not operator_nvrs:
+        # If this verb is run without operator NVRs, query Brew for all operator builds
+        operator_metas = [meta for meta in runtime.ordered_image_metas() if
+                          meta.enabled and meta.config['update-csv'] is not Missing]
+        runtime.images = [o.image_name for o in operator_metas]
```

Comment:
We're cloning all distgits but we don't need to, so set runtime.images to only we care about